packages/web/src/pages/explore-page/components/desktop/FeelingLuckySection.tsx

```diff
@@ -45,43 +51,46 @@ export const FeelingLuckySection = () => {
     [feelingLuckyUid, feelingLuckyTrackId, toggleFeelingLucky]
   )
 
-  if (!isLoading && feelingLuckyTrackId === 0) {
+  if (isError || (isSuccess && feelingLuckyTrackId === 0)) {
     return null
   }
 
+  // TODO: skeleton for loading
```

Comment:
in case you wanted to cover it in this pr

Reply:
Yeah I can do it here. It was just getting big 😅